src/installer/layouts.py

```diff
@@ -0,0 +1,83 @@
+__all__ = ["DistInfo"]
```

Comment:
why put this at the top, in my experience this is always at the end of file

Reply:
the name of this file is ``layouts`` but it really only handles ``DistInfo`` should we just name it ``dist_info.py`` then?

Reply:
Hmm, in mu experience `__all__` is always at the beginning (either at the very top or immediately below imports)

Reply:
I've never seen it at the beginning... I've certainly put it at the end always and goes well with the lint rule that imports are at the top, constants afterwards.

Reply:
As I said, my experience is otherwise. Django’s coding convention puts it right after imports, for example.

Some other examples for the standard library (not always a good source for coding conventions, but nonetheless):

* [Right after imports](https://github.com/python/cpython/blob/48b069a003ba6c684a9ba78493fbbec5e89f10b8/Lib/os.py). There are a lot of examples.
* [At the very beginning](https://github.com/python/cpython/blob/e42b705188271da108de42b55d9344642170aa2b/Lib/struct.py)

My quick search in stdlib also didn’t find any examples putting `__all__` at the end.

Reply:
I'm agreeable to put it after the imports, but before looks odd to me personally.

Reply:
seems it's also where stdlib generally puts it https://github.com/python/cpython/blob/master/Lib/os.py#L36 and https://github.com/python/cpython/blob/master/Lib/asyncio/__init__.py